tests/tc_ufcs_string_and_string_literal_completion/run.sh

```diff
@@ -0,0 +1,15 @@
+set -e
+set -u
+
+#TEST CASE 0
+SOURCE_FILE_0=file.d
+ACTUAL_FILE_NAME_0=actual_string_literal_test.txt
+EXPECTED_FILE_NAME_0=expected_string_literal_test.txt
```

Comment:
can you type the test filenames directly instead of using variable names or make the variable names actually match the file names? It's confusing that we suddenly use proper expected/actual names here, but the variables are still `*_0`, `*_1`, which makes it hard to mentally associate the correct test files with the tests.